### ISSUE TITLE
1.1.0 extract_1d testing

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#5865]
_Issue [JP-1971](https://jira.stsci.edu/browse/JP-1971) was created on JIRA by [Alicia Canipe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=acanipe):_

extract_1d version 1.1.0 updates are in this table: <https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=JPT&title=JP-1948+-<ins>Pipeline+Validation+Test+Epic</ins>-+2021-03-07>

Please test the extract_1d step.

**Comment by stscijgbot-jp** [comment]
_Comment by [Sarah Kendrew](https://jira.stsci.edu/secure/ViewProfile.jspa?name=skendrew) on [JIRA](https://jira.stsci.edu/browse/JP-1971?focusedCommentId=517625#comment-517625):_

Tested for MIRI LRS:
 * slit: [this notebook](https://jwst-validation-notebooks.stsci.edu/jwst_validation_notebooks/extract_1d/jwst_extract_1d_miri_test/extract_1d-spec2-miri-lrs-slit.html)
 * slitless: [this notebook](https://jwst-validation-notebooks.stsci.edu/jwst_validation_notebooks/extract_1d/jwst_extract_1d_miri_test/extract_1d-spec2-miri-lrs-slitless.html)

The test for SLIT was slightly out of range, for SLITLESS it passed. On looking into the issue for slit, I realized I was doing something slightly wrong. I am making a small change to both of these notebooks, so the method of testing is more accurate. This also makes all tests for B7.7 consistent with those of B7.6. 

Tl;dr: pipeline step is fine, my test needed a tweak.

**Comment by stscijgbot-jp** [comment]
_Comment by [Jo Taylor](https://jira.stsci.edu/secure/ViewProfile.jspa?name=jotaylor) on [JIRA](https://jira.stsci.edu/browse/JP-1971?focusedCommentId=518757#comment-518757):_

The NIRISS team has not tested this step. We are requesting an extension to complete this.

**Comment by stscijgbot-jp** [comment]
_Comment by [Alicia Canipe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=acanipe) on [JIRA](https://jira.stsci.edu/browse/JP-1971?focusedCommentId=518786#comment-518786):_

Tested for NIRCam WFSS with the following notebook (in the staging directory):
 * WFSS: ███████████████████████████████████████████████████████████████████████████████████████████████████

Current tests pass, the step is running as expected and generating 1D spectra.

**Comment by stscijgbot-jp** [comment]
_Comment by [Alicia Canipe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=acanipe) on [JIRA](https://jira.stsci.edu/browse/JP-1971?focusedCommentId=519274#comment-519274):_

Dry run is complete.
